src/Sulu/Bundle/AdminBundle/Resources/js/containers/MultiListOverlay/MultiListOverlay.js

```diff
@@ -67,13 +68,27 @@ class MultiListOverlay extends React.Component<Props> {
             USER_SETTINGS_KEY,
             observableOptions,
             options,
+            undefined,
             preloadSelectedItems ? preSelectedItems.map((preSelectedItem) => preSelectedItem.id) : undefined
         );
+
+        this.changeOptionsDisposer = reaction(
+            () => this.props.options,
```

Comment:
I think that this is running too often... You should be able to avoid that, by creating a computed getter for `options`.

Reply:
using `reaction` actually has the same effect as using a `computed` getter inside a `autorun`. the side effect of the `reaction` is only executed if the return value of the first function changes. in case of the `autorun`, the `autorun` is only executed if the value of the `computed` getter is changed (but not if one of the observables used inside the getter is changed).

see: https://mobx.js.org/refguide/reaction.html